OpenRA.Game/Network/LocalizedMessage.cs

```diff
@@ -130,7 +129,7 @@ public string Translate()
 					argumentDictionary.Add(argument.Key, new FluentString(argument.Value));
 			}
 
-			return Ui.Translate(Key, argumentDictionary);
+			return Game.ModData.Translation.GetFormattedMessage(Key, argumentDictionary);
```

Comment:
Can we please pass `ModData modData` in as an argument and use it here (method is only used from one place, so this isn't going to be a major refactoring)?